ena-submission/src/ena_deposition/create_assembly.py

```diff
@@ -134,11 +134,24 @@ def get_assembly_values_in_metadata(config: Config, metadata: dict[str, str]) ->
     return assembly_values
 
 
+def get_assembly_name(accession: str, version: str, test: bool = False) -> str:
```

Comment:
```suggestion
def make_assembly_name(accession: str, version: str, test: bool = False) -> str:
```

Get sounds like we're interacting with external services